lib/services/local/trust_toggle.go

```diff
@@ -0,0 +1,76 @@
+package local
+
+import (
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/services"
+
+	"github.com/gravitational/trace"
+)
+
+// TrustToggle is local implementation of TrustToggle service that uses
```

Comment:
why have a separate service for that, why not `Trust` service that manages trust? I would suggest to have `Trust` service that takes care of all trust, and not just turning on off, otherwise it's too fragmeneted